composer.json

```
@@ -11,7 +11,7 @@
         "api-platform/core": "^2.5",
         "babdev/pagerfanta-bundle": "^2.5",
         "beberlei/doctrineextensions": "^1.2",
-        "bolt/common": "^2.1.10",
+        "bolt/common": "^2.1.10 || ^3.0",
```

Let's make it 'just' 3.0, though. :-)

```suggestion
        "bolt/common": "^3.0",
```

---

well, already adjusted it now directly in code 😄